From cefc242dbfa40b781a1ddb32ff18565c7a0723e6 Mon Sep 17 00:00:00 2001
From: Pierre-Olivier Mercier <nemunaire@nemunai.re>
Date: Fri, 14 Oct 2016 18:07:32 +0200
Subject: [PATCH] Test with requests 2.8.1

---
 requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index 4661030..99da7b1 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1 +1 @@
-requests==2.11.1
+requests==2.8.1
-- 
2.7.3

