From 1d19abd597463c92d624bb69da374b98f9c821f4 Mon Sep 17 00:00:00 2001
From: Pierre-Olivier Mercier <nemunaire@nemunai.re>
Date: Mon, 27 Jan 2020 14:29:13 +0100
Subject: [PATCH] -

---
 libuuu/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libuuu/CMakeLists.txt b/libuuu/CMakeLists.txt
index 21ae944..38d6042 100644
--- a/libuuu/CMakeLists.txt
+++ b/libuuu/CMakeLists.txt
@@ -53,7 +53,7 @@ add_custom_command(
 	OUTPUT gitversion.h
 	PRE_BUILD
 	COMMAND mkdir -p ${generated_files_dir}
-	COMMAND sh -c 'cd ${CMAKE_CURRENT_SOURCE_DIR} && rm -f ${gitversion_h} && ./gen_ver.sh "${gitversion_h}.tmp" && mv -f "${gitversion_h}.tmp" "${gitversion_h}"'
+	COMMAND sh -c 'cd ${CMAKE_CURRENT_SOURCE_DIR} && rm -f ${gitversion_h} && ./gen_ver.sh "${gitversion_h}"'
 
 )
 include_directories(${generated_files_dir})
-- 
2.24.1

